management/daemon.py

```diff
@@ -126,27 +127,12 @@ def me():
 	# Is the caller authorized?
 	try:
 		email, privs = auth_service.authenticate(request, env)
-	except totp.MissingTokenError as e:
```

Comment:
I added this distinction to avoid the `/login` panel login flow logging at least one failed attempt towards fail2ban. Not sure about the best way to handle this.

Reply:
Hmm. I'm not sure that it shouldn't count. Good thought though.